ci/ci-pipeline.svg

```diff
@@ -0,0 +1,160 @@
+  <text x="101" y="277" text-anchor="start" font-family="Comic Sans MS, cursive, sans-serif" font-size="11" fill="#333">• CUDA version: 13.0.0 (build-time)</text>
+  <text x="502" y="262" text-anchor="start" font-family="Comic Sans MS, cursive, sans-serif" font-size="11" fill="#333">• Components: cuda-core, cuda-bindings,</text>
+  <text x="501" y="277" text-anchor="start" font-family="Comic Sans MS, cursive, sans-serif" font-size="11" fill="#333">  cuda-pathfinder, cuda-python</text>
+  
+  <!-- Rough arrow down -->
+  <path d="M451,292 Q449,305 450,312 M446,308 L450,314 L455,309" stroke="#333" stroke-width="2.5" fill="none" stroke-linejoin="round"/>
+  
+  <!-- Artifact Storage - hand-drawn style -->
+  <path d="M52,322 Q53,319 851,321 Q854,324 853,447 Q851,452 50,449 Q47,446 52,322 Z" 
+        fill="#fff3e0" stroke="#FF9800" stroke-width="2.5" stroke-linejoin="round"/>
+  <text x="451" y="342" text-anchor="middle" font-family="Comic Sans MS, cursive, sans-serif" font-size="14" font-weight="bold" fill="#F57C00">ARTIFACT STORAGE</text>
+  
+  <!-- Artifact types - rough rectangles -->
+  <path d="M122,357 Q123,354 398,356 Q402,358 401,433 Q399,437 121,436 Q118,433 122,357 Z" 
+        fill="white" stroke="#FF9800" stroke-width="2" stroke-linejoin="round"/>
+  <text x="261" y="377" text-anchor="middle" font-family="Comic Sans MS, cursive, sans-serif" font-size="12" font-weight="bold" fill="#333">GitHub Artifacts</text>
+  <text x="259" y="392" text-anchor="middle" font-family="Comic Sans MS, cursive, sans-serif" font-size="10" fill="#666">• Wheel files (.whl)</text>
+  <text x="261" y="404" text-anchor="middle" font-family="Comic Sans MS, cursive, sans-serif" font-size="10" fill="#666">• Test artifacts</text>
+  <text x="262" y="416" text-anchor="middle" font-family="Comic Sans MS, cursive, sans-serif" font-size="10" fill="#666">• Documentation</text>
+  <text x="260" y="430" text-anchor="middle" font-family="Comic Sans MS, cursive, sans-serif" font-size="9" fill="#666">(30-day retention)</text>
+  
+  <path d="M502,358 Q503,355 778,357 Q782,359 781,434 Q779,438 501,437 Q498,434 502,358 Z" 
+        fill="white" stroke="#FF9800" stroke-width="2" stroke-linejoin="round"/>
+  <text x="641" y="378" text-anchor="middle" font-family="Comic Sans MS, cursive, sans-serif" font-size="12" font-weight="bold" fill="#333">GitHub Cache</text>
+  <text x="639" y="402" text-anchor="middle" font-family="Comic Sans MS, cursive, sans-serif" font-size="10" fill="#666">• Mini CTK cache</text>
+  
+  <!-- Rough arrow down to Test Stage -->
+  <path d="M449,452 Q451,465 452,472 M447,468 L452,474 L457,469" stroke="#333" stroke-width="2.5" fill="none" stroke-linejoin="round"/>
+  
+  <!-- Artifact flow arrows -->
+  <!-- Arrow from GitHub Cache (mini CTK) back to Build Stage -->
+  <path d="M641,358 Q642,340 642,320 Q640,305 580,290 Q560,285 520,285" 
```

Comment:
The arrow should never overlap with the Build Stage box. In particular, the end point 285 is above the edge 290. Please make sure all elements of the arrow is below the box.